src/Language/PureScript/Names.hs

```diff
@@ -94,6 +99,9 @@ freshIdent name = GenIdent (Just name) <$> fresh
 freshIdent' :: MonadSupply m => m Ident
 freshIdent' = GenIdent Nothing <$> fresh
 
+unusedIdent :: Ident
```

Comment:
Why this indirection? You could just use the constructor.

Reply:
will do